lib/application.js

```diff
@@ -547,11 +547,16 @@ Origin.prototype.run = function () {
   // some modules are loaded async, so wait for them before actually starting the server
   this.once('modulesReady', function (app) {
     // Initialise the Polglot module so we can access language strings from the JSON
-    var phraseStrings = JSON.parse(fs.readFileSync(path.join(configuration.serverRoot, 'routes', 'lang', 'en.json'), {encoding: 'utf8'}));
-    app.polyglot = new polyglot({phrases: phraseStrings});
-    app.mailer = new Mailer();
-
-    app.startServer();
+    try {
+      var phraseStrings = JSON.parse(fs.readFileSync(path.join(configuration.serverRoot, 'routes', 'lang', 'en.json'), {encoding: 'utf8'}));
```

Comment:
I don't get it, that's been an issue since the first release.  That's why I wrapped it in a try-catch.